modules/bcftools/concat/meta.yml

```diff
@@ -0,0 +1,42 @@
+name: bcftools_concat
+description: Concatenate VCF files
+keywords:
+    - variant calling
+    - concat
+    - bcftools
+    - VCF
+
+tools:
+    - concat:
+        description: |
+          Concatenate VCF files.
+        homepage: http://samtools.github.io/bcftools/bcftools.html
+        documentation: http://www.htslib.org/doc/bcftools.html
+        doi: 10.1093/bioinformatics/btp352
+input:
+    - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test', single_end:false ]
+    - vcfs:
+        type: files
```

Comment:
Hmm, in this case I've followed the example of [bcftools/merge](https://github.com/nf-core/modules/blob/28b023e6f4d0d2745406d9dc6e38006882804e67/modules/bcftools/merge/main.nf#L22)

```nextflow
// main.nf
    input:
    tuple val(meta), path(vcfs), path(tbis)

```



And it's corresponding [meta.yml](https://github.com/nf-core/modules/blob/28b023e6f4d0d2745406d9dc6e38006882804e67/modules/bcftools/merge/meta.yml#L21) file mentions `vcf` only as `files`, instead of being a part of a map.

```nextflow
input:
    - meta:
        type: map
        description: |
          Groovy Map containing sample information
          e.g. [ id:'test', single_end:false ]
    - vcfs:
        type: files
```

Or have I missed the point here? 😅

Reply:
OK fair point. Go with the precendent ;).

I would argue the object going in is a `map` rather than a physical `file` object, but I don't think the `meta.yml` specs are standardised yet so safer to keep it consistent within the package.